Firefox: NPE in BiDiUti.readBiDiLogMessage after update to v7.12.0

Hi!

First of all, thanks for this awesome library!

Since the last update to v7.12.0, my log is filled with NPE errors (Firefox).
I can easily reproduce it when running e.g. the https://github.com/selenide-examples/google example repo with `./mvnw clean test -P firefox`.
The errors do not affect the test run, everything is fine and the tests are green, but it is really overloaded with these errors.

Firefox: 144.0.2
OS: MacOS Tahoe 26.2 (but I also see the same on CI, with runs with the [selenium/standalone-firefox:4](https://hub.docker.com/layers/selenium/standalone-firefox/4) docker image)
Java: Temurin-25.0.1

Log:
```log
➜ ./mvnw clean test -P firefox
WARNING: A restricted method in java.lang.System has been called
WARNING: java.lang.System::load has been called by org.fusesource.jansi.internal.JansiLoader in an unnamed module (file:/Users/lli/.m2/wrapper/dists/apache-maven-3.9.8/af622e91/lib/jansi-2.4.1.jar)
WARNING: Use --enable-native-access=ALL-UNNAMED to avoid a warning for callers in this module
WARNING: Restricted methods will be blocked in a future release unless native access is enabled

WARNING: A terminally deprecated method in sun.misc.Unsafe has been called
WARNING: sun.misc.Unsafe::objectFieldOffset has been called by com.google.common.util.concurrent.AbstractFuture$UnsafeAtomicHelper (file:/Users/lli/.m2/wrapper/dists/apache-maven-3.9.8/af622e91/lib/guava-33.2.1-jre.jar)
WARNING: Please consider reporting this to the maintainers of class com.google.common.util.concurrent.AbstractFuture$UnsafeAtomicHelper
WARNING: sun.misc.Unsafe::objectFieldOffset will be removed in a future release
[INFO] Scanning for projects...
[INFO]
[INFO] --------------------< org.selenide.examples:google >--------------------
[INFO] Building google 1.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- clean:3.2.0:clean (default-clean) @ google ---
[INFO] Deleting /Users/lli/GitWorkspace/coolstuff/google/target
[INFO]
[INFO] --- resources:3.3.1:resources (default-resources) @ google ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /Users/lli/GitWorkspace/coolstuff/google/src/main/resources
[INFO]
[INFO] --- compiler:3.13.0:compile (default-compile) @ google ---
[INFO] No sources to compile
[INFO]
[INFO] --- resources:3.3.1:testResources (default-testResources) @ google ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /Users/lli/GitWorkspace/coolstuff/google/src/test/resources
[INFO]
[INFO] --- compiler:3.13.0:testCompile (default-testCompile) @ google ---
[INFO] Recompiling the module because of changed source code.
[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[INFO] Compiling 13 source files with javac [debug target 25] to target/test-classes
[INFO]
[INFO] --- surefire:3.5.4:test (default-test) @ google ---
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-launcher/6.0.1/junit-platform-launcher-6.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-launcher/6.0.1/junit-platform-launcher-6.0.1.pom (3.2 kB at 16 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-launcher/6.0.1/junit-platform-launcher-6.0.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/platform/junit-platform-launcher/6.0.1/junit-platform-launcher-6.0.1.jar (246 kB at 5.4 MB/s)
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.selenide.examples.google.selenide_page_object_with_assertions.GoogleTest
[main] INFO com.codeborne.selenide.WebDriverThreadLocalContainer - No webdriver is bound to current thread: 3 - let's create a new webdriver
[main] INFO com.codeborne.selenide.webdriver.BrowserResizer - Set browser size to 1366x768
[main] INFO com.codeborne.selenide.webdriver.WebDriverFactory - Selenide 7.12.3 / Selenium 4.38.0 / firefox 144.0.2 mac
Exception in thread "BiDi Connection" Exception in thread "BiDi Connection" org.openqa.selenium.bidi.BiDiException: Unable to process: {"type":"event","method":"log.entryAdded","params":{"type":"javascript","level":"error","source":{"realm":"01e2be7b-79a4-491d-bf28-2303dc953fef","context":"304630e8-d6df-4bb1-9777-65690a4cdaf7"},"text":"Content-Security-Policy: The page’s settings blocked the loading of a resource (default-src) at https://duckduckgo.com/_next/static/chunks/7380-cb4d61a7fc4158b5.js because it violates the following directive: “default-src 'none'”","timestamp":1764772716191}}
Build info: version: '4.38.0', revision: '6b412e825c*'
System info: os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '26.1', java.version: '25.0.1'
Driver info: BiDi Connection
        at org.openqa.selenium.bidi.Connection$Listener.lambda$onText$0(Connection.java:279)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1090)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:614)
        at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.bidi.log.StackTrace.getCallFrames()" because the return value of "org.openqa.selenium.bidi.log.GenericLogEntry.getStackTrace()" is null
        at com.codeborne.selenide.impl.BiDiUti.readBiDiLogMessage(BiDiUti.java:71)
        at com.codeborne.selenide.impl.BiDiUti.readBiDiLogEntry(BiDiUti.java:67)
        at com.codeborne.selenide.impl.BiDiUti.lambda$readBiDiLogEntry$2(BiDiUti.java:62)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:214)
        at java.base/java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411)
        at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:803)
        at java.base/java.util.stream.ReferencePipeline$7$1FlatMap.accept(ReferencePipeline.java:293)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1024)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:635)
        at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:291)
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:652)
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:658)
        at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:663)
        at com.codeborne.selenide.impl.BiDiUti.readBiDiLogEntry(BiDiUti.java:63)
        at com.codeborne.selenide.impl.BiDiUti.lambda$collectBrowserLogs$0(BiDiUti.java:50)
        at org.openqa.selenium.bidi.Connection.lambda$handleEventResponse$7(Connection.java:387)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:186)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:197)
        at java.base/java.util.HashMap$EntrySpliterator.forEachRemaining(HashMap.java:1859)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
        at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:153)
        at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:176)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
        at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:632)
        at org.openqa.selenium.bidi.Connection.handleEventResponse(Connection.java:367)
        at org.openqa.selenium.bidi.Connection.handle(Connection.java:304)
        at org.openqa.selenium.bidi.Connection$Listener.lambda$onText$0(Connection.java:277)
        ... 3 more
```

--- a/src/main/java/com/codeborne/selenide/impl/BiDiUti.java
+++ b/src/main/java/com/codeborne/selenide/impl/BiDiUti.java
@@ -68,7 +68,7 @@ public class BiDiUti {
   }
 
   private static String readBiDiLogMessage(GenericLogEntry log) {
-    if (log.getStackTrace().getCallFrames().isEmpty()) {
+    if (log.getStackTrace() == null || log.getStackTrace().getCallFrames().isEmpty()) {
       return log.getText();
     }
     StackFrame frame = log.getStackTrace().getCallFrames().get(0);

